python_modules/libraries/dagster-airbyte/dagster_airbyte/translator.py

```diff
@@ -1,12 +1,23 @@
-from typing import Any, Mapping, NamedTuple, Optional
+from typing import Any, List, Mapping, NamedTuple, Optional, Sequence
 
 from dagster._annotations import experimental
 from dagster._core.definitions.asset_spec import AssetSpec
 from dagster._record import record
 from dagster._serdes.serdes import whitelist_for_serdes
+from dagster._utils.cached_method import cached_method
 
+from dagster_airbyte.utils import get_airbyte_connection_table_name
 
-class AirbyteConnectionTableProps(NamedTuple): ...
+
+class AirbyteConnectionTableProps(NamedTuple):
```

Comment:
We could also name this AirbyteConnectionStreamProps if we want to keep as close as possible to Airbyte's ontology. Open to feedback.